src/views/static/people/people-view.js

```diff
@@ -0,0 +1,35 @@
+import { customElement, html, LitElement } from 'lit-element';
+
+import styles from './../static-view.styles';
+
+@customElement('people-view')
+export class PeopleView extends LitElement {
+  static get styles() {
+    return [styles];
+  }
+
+  render() {
+    return html`
+      <div id="people">
+        <div class="main-border">
+          <div class="main-content">
+            <p class="construction">PEOPLE PAGE</p>
```

Comment:
@Wirnhier Great job on removing the redundant code! These few `construction` paragraphs are still duplicated though. Can we create a separate component for this? It can be called `<construction-warning/>` for example.

Additionally, can we hide this warning for screens larger than 1000px using media queries? We don't need to do it in this PR, but it would be nice to have!

Reply:
I don't know if we need it at all. For me that's part of the content, it comes later.

Reply:
if the content is not needed, can we delete it? We can use `return null;` instead of all the repeated HTML.